### ISSUE TITLE
Unable to add content to a current brew, unable to save new ones?

### ISSUE THREAD

**Issue by bumblemute** [stolksdorf/homebrewery#155]
If you are reporting a bug, please fill out the details. If you don't I will most likely just close the issue asking for more detail.

---

**Browser Type/Version**: Chrome,  51.0.2704.103 m

**Operating System**: Windows 8.1

**Issue Description**: Cannot add any more content to my latest brew, if I add even another line, it fails to save, stating something's gone wrong. I can't make a new brew either, as when I hit save, nothing happens.

**Brew code to reproduce**: n/a

**Comment by kmcmanus** [comment]
I'm also seeing this, in Chrome 51.0.2704.106 (64-bit) on Linux Mint. I'm seeing the `/api` call return a 503 (Service Unavailable) error when trying to save a new document.

**Comment by stolksdorf** [comment]
Figured out the issue. There will be a bit of downtime to fix it. Too many homunculi in the DB. I'll update this thread when I have estimates.

**Comment by stolksdorf** [comment]
Fixed! Whew, that was incredibly stressful. There was apparently a random uptick in popularity of the Homebrewery in the last 48hrs-ish which blew through all my database quotas.

I had to provision a bunch of more resources, but had to move over to a different infrastructure to do so, which involved making back-ups and taking down the site while the migration occurred.

Everything is good now, very little was lost, and we should be good for a _loooooooong_ time.

Time to play Overwatch and have a beer. Thanks for dealing with this mess!
